<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="wyloguj">
    <property name="geometry">
     <rect>
      <x>680</x>
      <y>20</y>
      <width>90</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Wyloguj</string>
    </property>
   </widget>
   <widget class="QLabel" name="kto_zalogowany">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>171</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Kierownik</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTabWidget" name="zakladki_gora">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>791</width>
      <height>461</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>16</height>
     </size>
    </property>
    <widget class="QWidget" name="przedmioty">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <attribute name="title">
      <string>Przedmioty</string>
     </attribute>
     <widget class="QCheckBox" name="czyUszkodzone">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>20</y>
        <width>111</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>uszkodzone</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="czySprawne">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>50</y>
        <width>109</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>sprawne</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="czyZgubione">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>80</y>
        <width>111</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>zgubione</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="czyZniszczone">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>110</y>
        <width>111</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>zniszczone</string>
      </property>
     </widget>
     <widget class="QTabWidget" name="zakladki_dol">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>160</y>
        <width>791</width>
        <height>281</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="maszyny">
       <attribute name="title">
        <string>Maszyny</string>
       </attribute>
       <widget class="QPushButton" name="zamow_maszyny">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>10</y>
          <width>241</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Zamów</string>
        </property>
       </widget>
       <widget class="QTableView" name="maszyny_table">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>50</y>
          <width>781</width>
          <height>192</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="dodaj_maszyne">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>241</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Dodaj maszynę</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="skrzynki">
       <attribute name="title">
        <string>Skrzynki</string>
       </attribute>
       <widget class="QTableView" name="skrzynki_table">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>50</y>
          <width>781</width>
          <height>192</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="dodaj_skrzynke">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>241</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Dodaj skrzynkę</string>
        </property>
       </widget>
       <widget class="QPushButton" name="zamow_skrzynki">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>10</y>
          <width>241</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Zamów</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pokaz_zawartosc">
        <property name="geometry">
         <rect>
          <x>530</x>
          <y>10</y>
          <width>241</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Pokaż zawartość</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="elektronarzedzia">
       <attribute name="title">
        <string>Elektronarzędzia</string>
       </attribute>
       <widget class="QTableView" name="elektronarzedzia_table">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>50</y>
          <width>781</width>
          <height>192</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="dodaj_elektronarzedzie">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>241</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Dodaj elektronarzędzie</string>
        </property>
       </widget>
       <widget class="QPushButton" name="zamow_elektronarzedzie">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>10</y>
          <width>241</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Zamów</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="akcesoria">
       <attribute name="title">
        <string>Akcesoria</string>
       </attribute>
       <widget class="QTableView" name="akcesoria_table">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>50</y>
          <width>781</width>
          <height>192</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="dodaj_akcesorium">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>241</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Dodaj akcesorium</string>
        </property>
       </widget>
       <widget class="QPushButton" name="zamow_akcesorium">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>10</y>
          <width>241</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Zamów</string>
        </property>
       </widget>
       <widget class="QPushButton" name="dodaj_akcesorium_doskrzynki">
        <property name="geometry">
         <rect>
          <x>530</x>
          <y>10</y>
          <width>241</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Dodaj do skrzynki</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="narzedzia">
       <attribute name="title">
        <string>Narzędzia</string>
       </attribute>
       <widget class="QTableView" name="narzedzia_table">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>50</y>
          <width>781</width>
          <height>192</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="dodaj_narzedzie">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>241</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Dodaj narzędzie</string>
        </property>
       </widget>
       <widget class="QPushButton" name="zamow_narzedzie">
        <property name="geometry">
         <rect>
          <x>270</x>
          <y>10</y>
          <width>241</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Zamów</string>
        </property>
       </widget>
       <widget class="QPushButton" name="dodaj_narzedzie_doskrzynki">
        <property name="geometry">
         <rect>
          <x>530</x>
          <y>10</y>
          <width>241</width>
          <height>31</height>
         </rect>
        </property>
        <property name="text">
         <string>Dodaj do skrzynki</string>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QLineEdit" name="nazwa">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>20</y>
        <width>211</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Nazwa</string>
      </property>
     </widget>
     <widget class="QPushButton" name="wyszukaj_opis">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>20</y>
        <width>90</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Wyszukaj</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="producent">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>60</y>
        <width>211</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Producent</string>
      </property>
     </widget>
     <widget class="QPushButton" name="wyszukaj_producent">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>60</y>
        <width>90</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Wyszukaj</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="dzialy">
     <attribute name="title">
      <string>Działy</string>
     </attribute>
     <widget class="QPushButton" name="dodaj_pracownika">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Dodaj pracownika</string>
      </property>
     </widget>
     <widget class="QPushButton" name="zwolnij_pracownika">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>10</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Zwolnij pracownika</string>
      </property>
     </widget>
     <widget class="QPushButton" name="dodaj_zgloszenie">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>10</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Dodaj zgłoszenie</string>
      </property>
     </widget>
     <widget class="QListView" name="dzialy_list">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>121</width>
        <height>341</height>
       </rect>
      </property>
     </widget>
     <widget class="QTabWidget" name="tabWidget">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>90</y>
        <width>651</width>
        <height>341</height>
       </rect>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="pracownicy">
       <attribute name="title">
        <string>Pracownicy</string>
       </attribute>
       <widget class="QTableView" name="tableView">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>641</width>
          <height>341</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="skrzynki_2">
       <attribute name="title">
        <string>Skrzynki</string>
       </attribute>
       <widget class="QTableView" name="tableView_2">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>641</width>
          <height>341</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="maszyny_2">
       <attribute name="title">
        <string>Maszyny</string>
       </attribute>
       <widget class="QTableView" name="tableView_3">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>641</width>
          <height>341</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
     <widget class="QPushButton" name="przenies_skrzynke">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Przenieś skrzynkę</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pokaz_zawartosc_2">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>50</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Pokaż zawartość skrzynki</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3lista_zakupow">
     <attribute name="title">
      <string>Lista Zakupów</string>
     </attribute>
     <widget class="QTableView" name="listazakupow_table">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>50</y>
        <width>781</width>
        <height>381</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="przenies_dozrealizowanych">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Przenieś do zrealizowanych</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="raport1">
     <attribute name="title">
      <string>Raport 1</string>
     </attribute>
     <widget class="QPushButton" name="generuj_raport_1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Generuj raport</string>
      </property>
     </widget>
     <widget class="QLabel" name="od_2">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>10</y>
        <width>21</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Od</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="od_date_2">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>10</y>
        <width>110</width>
        <height>31</height>
       </rect>
      </property>
      <property name="dateTime">
       <datetime>
        <hour>0</hour>
        <minute>0</minute>
        <second>0</second>
        <year>2017</year>
        <month>1</month>
        <day>1</day>
       </datetime>
      </property>
     </widget>
     <widget class="QLabel" name="do_3">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>10</y>
        <width>21</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Do</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="do_date_2">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>10</y>
        <width>110</width>
        <height>31</height>
       </rect>
      </property>
      <property name="dateTime">
       <datetime>
        <hour>0</hour>
        <minute>0</minute>
        <second>0</second>
        <year>2018</year>
        <month>1</month>
        <day>1</day>
       </datetime>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>50</y>
        <width>791</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>60</y>
        <width>781</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Dział z największą ilością zagubionych i uszkodzonych przedmiotów&lt;/p&gt;&lt;p&gt;z wykazem pracowników&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="Line" name="line_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>110</y>
        <width>791</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="nazwa_dzialu_report1">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>120</y>
        <width>141</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Nazwa działu</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QTableView" name="raport1_table">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>160</y>
        <width>781</width>
        <height>261</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="raport2">
     <attribute name="title">
      <string>Raport 2</string>
     </attribute>
     <widget class="QLabel" name="od_3">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>10</y>
        <width>21</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Od</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>60</y>
        <width>781</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Lista materiałów eksploatacyjnych wg. zużycia ilościowego&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QTableView" name="raport2_table">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>120</y>
        <width>781</width>
        <height>301</height>
       </rect>
      </property>
     </widget>
     <widget class="Line" name="line_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>50</y>
        <width>791</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="do_4">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>10</y>
        <width>21</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Do</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="do_date_3">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>10</y>
        <width>110</width>
        <height>31</height>
       </rect>
      </property>
      <property name="dateTime">
       <datetime>
        <hour>0</hour>
        <minute>0</minute>
        <second>0</second>
        <year>2018</year>
        <month>1</month>
        <day>1</day>
       </datetime>
      </property>
     </widget>
     <widget class="QPushButton" name="generuj_raport_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Generuj raport</string>
      </property>
     </widget>
     <widget class="QDateEdit" name="od_date_3">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>10</y>
        <width>110</width>
        <height>31</height>
       </rect>
      </property>
      <property name="dateTime">
       <datetime>
        <hour>0</hour>
        <minute>0</minute>
        <second>0</second>
        <year>2017</year>
        <month>1</month>
        <day>1</day>
       </datetime>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
